internal/manifest/v1beta1/spec_resolver.go

```diff
@@ -83,7 +83,7 @@ func (m *ManifestSpecResolver) Spec(ctx context.Context, obj declarative.Object)
 	case v1beta1.HelmChartType:
 		mode = declarative.RenderModeHelm
```

Comment:
Why we still support Helm and Kostomize? If the purpose for this issue is remove helm chart support?

Reply:
I am not sure either, TBH. As far as I understand, the last time we discussed the scope of the change, we agreed not to remove other renderers (as we are not live yet, and we can not be sure that some partial Kustomize or Helm support might be needed). But if you think we can drop it, that's also valid.

Reply:
As we clarified, helm and kostomize support should be completely removed.